### PR TITLE
Add new ubuntu versions

### PR DESCRIPTION
**Pull request by erikjohnston**
c.f. https://github.com/element-hq/synapse/pull/17824

### CODE REVIEW

**Review by sandhose — APPROVED** [review]
I don't know much about debian repos, but this looks reasonable

File: debian/conf/distributions [hunk 1]
```diff
@@ -149,3 +149,19 @@ Architectures: amd64 i386 source
 Components: main prerelease
 SignWith: F473DD4473365DE1
 Tracking: minimal
+
```
Comment (anoadragon453) [inline debian/conf/distributions:152]: Shouldn't we remove `lunar` and `mantic`, to align with https://github.com/element-hq/synapse/pull/17824/files?

**Comment by anoadragon453** [comment]
https://github.com/matrix-org/pkg-repo-configs/blob/2eb163fce4880609b32115f83b6db27288716202/debian/conf/incoming#L4 should also be updated.

File: debian/conf/distributions (hunk 1 above)
Comment (erikjohnston) [reply debian/conf/distributions:152]: We have kept all the other old ubuntu versions. I think probably so things don't break for people running them

**Review by anoadragon453 — APPROVED** [review]

**Pull request merged**